pkg/new-ui/v1beta1/frontend/src/app/enumerations/metrics-collector.ts

```diff
@@ -0,0 +1,9 @@
+export enum CollectorKind {
+  STDOUT = 'StdOut',
+  FILE = 'File',
+  TFEVENT = 'TensorFlowEvent',
+  PROMETHEUS = 'PrometheusMetric',
+  CUSTOM = 'Custom',
+  NONE = 'None',
+  VOLUME = 'metrics-volume',
```

Comment:
Do we need this @kimwnasptd ? We don't have such MC kind.
```suggestion
```

Reply:
I included it because I found it's definition in the go types https://github.com/kubeflow/katib/blob/master/pkg/apis/controller/common/v1beta1/common_types.go#L217

If it's not yet functional/implemented then sure I'll remove it

Reply:
It's just a const which is used here: https://github.com/kubeflow/katib/blob/master/pkg/webhook/v1beta1/pod/utils.go#L215.
Not a MC type.

Reply:
ACK, removing it